frontend/wikiPages/ca/wiki/Welcome

```diff
@@ -0,0 +1,83 @@
+<h1>Benvingut</h1>
+
+<div class="content">
+
+<p>Heu arribat a la Pàgina Web de <b>PhiloBiblon </b>una base de dades biobibliogràfica sobre textos antics escrits en diverses llengües romàniques en la península Ibèrica en l'edat mitjana i inicis del Renaixement, informació accessible lliurement a través d'Internet.</p>
+
+<p>Una cerca a <a href="ca/search/texid/query">Obra</a> ens proporciona una llista de registres que contenen els termes inclosos a la cerca. F
+
+eu clic en l’obra que us interessa i obtindreu una llista dels seus testimonis presentats en ordre cronològic.
+
+Cadascun dels testimonis està vinculat al  manuscrit o a l’edició impresa que el conté.</p><p>
+
+</p><p>Una cerca a <a href="ca/search/manid/query">MsEd</a> ens proporciona una llista de registres que contenen els termes inclosos a la cerca.
+
+Feu clic al manuscrit o a l’edició impresa que us interessa i trobareu una descripció externa de l’exemplar,
+
+seguida d'una descripció del seu contingut, amb les diverses obres que transmet l’exemplar ordenades segons la seva aparició al volum.</p>
+
+<p>PhiloBiblon (nom que es pren de la descripció que feu Richard de Bury al s. XIV d'una biblioteca ideal) és el corpus de dades catalogat en el projecte Philobiblon: les fonts primàries dels textos en llengua romanç medievals o dels inicis de l'època moderna a la península Ibèrica. és també un sistema de gestió de dades dissenyat per catalogar les fonts primàries incloses en ella, tant impreses com manuscrites, a més d'altres còpies o exemplars més tardans. En la base de dades també es trobaran referències a les persones involucrades en el procés de producció i/o transmissió literària, i també a les institucions on es conserven els materials analitzats, juntament amb la bibliografia secundària més rellevant per a l'estudi de les fonts, persones, llocs i institucions esmentades a la base de dades. </p>
+
+<p>Si desitgeu conèixer les darreres novetats, consulteu el <b>PhiloBLOG</b>, a la banda esquerra d'aquesta pàgina, o seguiu PhiloBiblon a <b>Facebook</b> i <b>Twitter</b>.</p>
+
+<p>PhiloBiblon consisteix en quatre bibliografies diferents, cadascuna de les quals té la seva própia pàgina principal: </p>
+
+<p><a href="ca/wiki/Beta"><b>BETA</b></a><b> / Bibliografía Española de Textos Antiguos</b><br>
```

Comment:
We need to modify all the wiki pages links to the wiki format.
For example in this case:
```
<a href="ca/wiki/Beta"><b>BETA</b></a>
```
to:
```
[[ca_Beta|<b>BETA</b>]]
```
In this way the links will work in Wikibase and in PB UI.